examples/pkg/clusterclient/clientset.go

```diff
@@ -2,7 +2,7 @@
 // +build !ignore_autogenerated
 
 /*
-Copyright The KCP Authors.
+Copyright 2022 The KCP Authors.
```

Comment:
The year must not be here - did this regress?

Reply:
Right, the header file in testdata had to be updated.